cmd/adapter/adapter.go

```diff
@@ -320,7 +320,7 @@ func main() {
 		cmd.OpenAPIConfig.Info.Version = "1.0.0"
 	}
 
-	if cmd.OpenAPIV3Config == nil && utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3) {
+	if cmd.OpenAPIV3Config == nil { // default on utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3) {
```

Comment:
Will handle this gracefully later.

Now want to see whether E2E tests can pass

Reply:
cc @dgrisonnet@google.com, any suggestions here?

Could we remove `utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3)` directly?